### DISCUSSION TITLE
2024/06/02/The-Messy-Math/

### DISCUSSION THREAD

**Discussion by giscus** [MrWillCom/MrWillCom.github.io#11] [Announcements]
# 2024/06/02/The-Messy-Math/

https://blog.mrwillcom.com/2024/06/02/The-Messy-Math/

**Comment by MrWillCom** [comment]
[王垠的《数学和编程》一文](https://www.yinwang.org/blog-cn/2015/07/04/math)中也讨论了相关的问题，可供参考。

**Reply by MrWillCom** [reply]
这篇文章现在的地址在：[https://www.yinwang.org/posts/math](https://www.yinwang.org/posts/math)
